Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -384,7 +384,7 @@ void CEXIETHERNET::BuiltInBBAInterface::HandleTCPFrame(const Common::TCPPacket&
     if (size > 0)
     {
       // only if contain data
-      if (static_cast<int>(this_seq - ref->ack_num) >= 0 && data.size() >= size)
+      if (static_cast<int>(this_seq - ref->ack_num) >= 0 && static_cast<int>(data.size()) >= size)
```

Comment:
This has already checked `size > 0` two lines above, so why are you casting the unsigned type to signed for the comparison instead of making the already known positive value unsigned?